content/en/serverless/azure_functions/_index.md

```diff
@@ -56,43 +51,117 @@ This page explains how to collect traces, trace metrics, runtime metrics, and cu
 {{< /programming-lang >}}
 {{< programming-lang lang="java" >}}
 1. **Install dependencies**. Download the Datadog JARs and deploy them with your function:
    ```bash
    wget -O dd-java-agent.jar 'https://dtdg.co/latest-java-tracer'
    wget -O dd-serverless-compat-java-agent.jar 'https://dtdg.co/latest-serverless-compat-java-agent'
    ```
-   See Datadog's [Maven Repository][4] for any specific version of the Datadog Serverless Compatibility Layer.
-
-   To use [automatic instrumentation][1], you must use `dd-java-agent` v1.48.0+.
+   See Datadog's [Maven Repository][3] for any specific version of the Datadog Serverless Compatibility Layer.
 
    Datadog recommends regularly upgrading to the latest versions of both `dd-serverless-compat-java-agent` and `dd-java-agent` to ensure you have access to enhancements and bug fixes.
 
-2. **Start the Datadog serverless compatibility layer and initialize the Java tracer**. Add the following `-javaagent` arguments to the JVM options.:
+2. **Start the Datadog Serverless Compatibility Layer and initialize the Datadog Java tracer**. Add the following `-javaagent` arguments to the JVM options.:
 
    ```bash
    -javaagent:/path/to/dd-serverless-compat-java-agent.jar -javaagent:/path/to/dd-java-agent.jar
    ```
 
-   **Note**: the environment variable to set JVM options depends on the hosting plan (example, Consumption, Elastic Premium, Dedicated). See [Azure Functions Java developer guide][5] for more details on the appropriate environment variable for your hosting plan.
+   **Note**: the environment variable to set JVM options depends on the hosting plan (example, Consumption, Elastic Premium, Dedicated). See [Azure Functions Java developer guide][4] for more details on the appropriate environment variable for your hosting plan.
+
+3. (Optional) **Enable runtime metrics**. See [Java Runtime Metrics][1].
+
+4. (Optional) **Enable custom metrics**. See [Metric Submission: DogStatsD][2].
+
+[1]: /tracing/metrics/runtime_metrics/?tab=java#environment-variables
+[2]: /metrics/custom_metrics/dogstatsd_metrics_submission/?tab=java
+[3]: https://repo1.maven.org/maven2/com/datadoghq/dd-serverless-compat-java-agent/
+[4]: https://learn.microsoft.com/en-us/azure/azure-functions/functions-reference-java?tabs=bash%2Cconsumption#customize-jvm
+{{< /programming-lang >}}
+{{< programming-lang lang="dotnet" >}}
+1. **Install dependencies**. Run the following commands:
+   ```shell
+   dotnet package add Datadog.Azure.Functions
+   ```
+
+   Datadog recommends regularly upgrading to the latest version of `Datadog.AzureFunctions` to ensure you have access to enhancements and bug fixes.
+
+2. **Start the Datadog Serverless Compatibility Layer**. Add the following lines to your main application entry point file:
+   If your Azure Function app uses the Isolated Worker model
+   ```csharp
+   using Datadog.Serverless;
+   ...
+   Datadog.Serverless.CompatibilityLayer.Start();
```

Comment:
We don't need `using Datadog.Serverless` if using the full namespace below (which is simpler).

```suggestion
   Datadog.Serverless.CompatibilityLayer.Start();
```